GraphQL/Navigating schemas, types, fetching requests, using mutations & subscriptions/proposal.md

```diff
@@ -0,0 +1,36 @@
+# TEMPLATE
+
+## :fire: Do not edit this file - copy the template and create your own file.
+
+**[Step-By-Step Technical Blog Guide](https://hq.bitproject.org/how-to-write-a-technical-blog/)**
+
+### :pushpin: Step 1
+**TITLE:**    
+Navigating schemas/types/fetching requests, using mutations/subscriptions
+
+**TOPIC:**    
+GraphQL
+
+**DESCRIPTION (5-7+ sentences):**    
+
+GraphQL, a language for APIs, was first developed by Facebook. After it was introduced, it became extremely popular as some considered it a better API than rest API because underfetching and overfetching data no longer became a problem. GraphQL allows us to specify the root field and the payload from the client side, giving greater flexibility to manipulate the data we want to request from the server. Furthermore, GraphQL has special features of allowing the client to update data, commonly known as mutations, as well as receive real-time data, the concept of subscriptions. These are the core concepts I will be covering within my blog. 
```

Comment:
I'm wondering whether "rest" in "rest API" is an acronym of some kind? This would help the type of API you mention here. Otherwise, the description looks good.